sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -1332,14 +1325,14 @@
           {
             "name": "jobType",
             "in": "query",
-            "description": "Job Type.",
+            "description": "The job type.",
```

Comment:
This may be more of a question for someone on the service side, but what are the possible values here?

Reply:
Since it is a string, not an enum, it would be helpful to clarify the possible values

Reply:
I believe two possible values:

scheduleUpdateTwin: A job used to update a set of desired properties or tags.
scheduleDeviceMethod: A job used to invoke a device method on a set of device twins.

This is from this doc: https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-jobs#jobs-to-update-device-twin-properties

Reply:
But if they start supporting new types, they will have to update this comment. Do you think it could eventually lead to gaps? Wondering if it is better to not add it in comments. I'm guessing when the service returns a 400, they will tell you in the exception about the allowed values.